lib/v2/xys/new.js

```diff
@@ -0,0 +1,72 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const iconv = require('iconv-lite');
+const path = require('path');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+
+module.exports = async (ctx) => {
+    const rootUrl = 'http://www.xys.org';
+    const currentUrl = `${rootUrl}/new.html`;
+
+    const response = await got({
+        method: 'get',
+        url: currentUrl,
+        responseType: 'buffer',
+    });
+
+    // 转码
+    const data = iconv.decode(response.data, 'gb2312');
+
+    const $ = cheerio.load(data);
+
+    let items = $('li a')
+        .slice(4, ctx.query.limit ? parseInt(ctx.query.limit) : 30)
+        .toArray()
+        .map((item) => {
+            item = $(item);
+            let link = item.attr('href');
+            /^https?:\/\//.test(link) || (link = rootUrl + '/' + link.replace(/^\//, ''));
+            let date = item.parent().text().trim().slice(0, 8);
+            date = parseDate(date, 'YY.MM.DD');
+            return {
+                title: item.text(),
+                link,
+                pubDate: date,
+            };
+        });
+
+    items = await Promise.all(
+        items.map((item) =>
```

Comment:
```suggestion
        items.filter((item) => !item.link.endsWith('.zip')).map((item) =>
```
so we won't display the zip buffer in the RSS feed or you can filter them out when you defined `items`

Reply:
good point!